### ISSUE TITLE
Is it possible to show specific div as a Print Preview?

### ISSUE THREAD

**Issue by billytsr** [etimbo/jquery-print-preview-plugin#15]
Is it possible to show specific div as a Print Preview instead of showing complete Page?I'm preferring dynamic div through Javascript or jQuery. Thanks.

**Comment by srnimesh** [comment]
yes, im also have the above mentioned doubt. I t takes the whole preview of a page. Is any solution for this?
               Thanks

**Comment by jurgen-74** [comment]
I have a solution but it is not really satisfying:
`<style>.no-print{display: none;}</style>`

`<div class="no-print" id="001">`

`<script type="text/javascript" >$("#001").hover(function() {  $(this).removeClass("no-print");}, function() {$(this).addClass("no-print");});</script>`

`HERE YOUR CONTENT OF THE DIV AND YOUR PRINT BUTTON:`

`<a class="print-preview" href="#" >print this part only</a>`

`</div>`

The trick here is that when the user goes to the print button his mouse hovers over that area. This action removes the no-print class and makes the area visible for the plugin. You can see an example here: [http://www.apologeet.nl/faq/tegenstrijdigheden](url)
